src/main/java/seedu/address/model/show/Status.java

```diff
@@ -21,9 +23,15 @@ public String toString() {
      * @return true if the show status is completed.
      */
     public Boolean isCompleted() {
-        if (status.equals("completed")) {
-            return true;
-        }
-        return false;
+        return status.equals(COMPLETED);
     }
+
+    /**
+     * Returns true if a given string is a valid status.
+     */
+    public static boolean isValidStatus(String test) {
+        String stest = test.toLowerCase().trim();
+        return (stest.equals(COMPLETED) || stest.equals(WATCHING));
+    }
+
 }
```

Comment:
Regarding Status, shall we first have a discussion on the change of implementation first?

Reply:
What do you suggest?

Reply:
We've started a discussion thread under "Issues". Let's discuss there instead!

Reply:
Updated